packages/shopify-api/adapters/node/index.ts

```diff
@@ -8,18 +10,19 @@ import {
   setAbstractConvertHeadersFunc,
   setAbstractRuntimeString,
   setCrypto,
+  AbstractFetchFunc,
 } from '../../runtime';
 
 import {
-  nodeFetch,
   nodeConvertRequest,
   nodeConvertIncomingResponse,
   nodeConvertAndSendResponse,
   nodeConvertAndSetHeaders,
   nodeRuntimeString,
 } from './adapter';
 
-setAbstractFetchFunc(nodeFetch);
+// For the purposes of this package, fetch correctly implements everything we need
+setAbstractFetchFunc(fetch as any as AbstractFetchFunc);
```

Comment:
Is there anyway to improve the type of `AbstractFetchFunc` so that we don't need to cast as any?